### PR TITLE
docs: update the structure of the documentation

### PR DESCRIPTION
**Pull request by profanis**
In this PR I am suggesting a new structure for the documentation. Since it's hard to visualize the structure by reading the MD files, I am attaching these images to make it easier.

To change the outline of the documentation, I used gitbook's CMS. Even though I made changes only on the outline, the gitbook did a normalization on some pages and as such, this PR also has some page content changes. 

menu_1
menu_2
menu_3

### CODE REVIEW

**Review by Carniatto — COMMENTED** [review]
I'd promote the discord instead of slack since we are changing the Readme

File: docs/README.md [hunk 1]
```diff
@@ -1,64 +1,49 @@
...
 
-## Individuals
+### Individuals
 
 [![Backers](https://opencollective.com/ngxs/backers.svg?width=890)](https://opencollective.com/ngxs#backers)
 
-# Contributors
+## Contributors
 
 Thanks to all our [contributors](https://github.com/ngxs/store/graphs/contributors)! Open source does not work without you!
```
Comment (Carniatto) [inline docs/README.md:48]: Could we add the contributors with small pictures? this would add some visibility of our contributors

Comment (profanis) [reply docs/README.md:48]: I toggled the rich diff feature to see the size of the contributors' avatars. I am not sure how to make them smaller. I am afraid though, that a smaller avatar won't be sufficient. :/ 


image

**Comment by profanis** [comment]
> I'd promote the discord instead of slack since we are changing the Readme

@Carniatto  I tried the discord shield in the readme file and I got "Widget Disabled" 
image

The discord admin should enable the widget setting on the server.  This page explains how https://shields.io/badges/discord

**Comment by Carniatto** [comment]
@profanis Sorry for the super long delay, the server widget has been activated. It should work now!

**Comment by markwhitfeld** [comment]
@profanis I see that there are many conflicts because there has been a lot of updates to the docs.
I hope that the conflict resolution is not too crazy. Unless it is better to use the current state of the docs, and redo the changes you did (...if it was mainly file movements).
Please DM me once done so that we can merge and avoid more conflicts 😉 🙏

**Comment by profanis** [comment]
> @profanis I see that there are many conflicts because there has been a lot of updates to the docs. I hope that the conflict resolution is not too crazy. Unless it is better to use the current state of the docs, and redo the changes you did (...if it was mainly file movements). Please DM me once done so that we can merge and avoid more conflicts 😉 🙏

I resolved the conflicts but I will do a cross-check to make sure everything is in place

**Comment by markwhitfeld** [comment]
@profanis I had a look through, and many of the recent PRs docs changes have been lost.
You may need to check through each of these PRs.

**Comment by markwhitfeld** [comment]
@profanis I think that a better approach would be to go back to [the branch head](https://github.com/ngxs/store/commit/8e2056fd8bfea9e86d0b74299715e798d2a13285) you had before you force pushed, and then cherry pick each of the changes that have landed on master into your branch, one by one. This will allow you to decide what should be done for each commit.

**Comment by profanis** [comment]
> @profanis I think that a better approach would be to go back to [the branch head](https://github.com/ngxs/store/commit/8e2056fd8bfea9e86d0b74299715e798d2a13285) you had before you force pushed, and then cherry pick each of the changes that have landed on master into your branch, one by one. This will allow you to decide what should be done for each commit.

Thanks for checking at this @markwhitfeld 
It had that many conflicts and made it hard. I am closing this PR and opening this https://github.com/ngxs/store/pull/2139

**Pull request closed without merging**